anndata/tests/test_dask.py

```diff
@@ -3,12 +3,53 @@
 """
 import anndata as ad
 import pandas as pd
-
+from anndata._core.anndata import AnnData
 import pytest
+from anndata.tests.helpers import (
+    darr_from_arr,
+    gen_adata_dask_args,
+    gen_adata,
+    assert_equal,
+    asarray,
+)
+from anndata.compat import DaskArray
 
 pytest.importorskip("dask.array")
 
 
+@pytest.fixture(
+    params=[
+        [(2000, 1000), (100, 100)],
+        [(200, 100), (100, 100)],
+        [(200, 100), (100, 100)],
+        [(20, 10), (1, 1)],
+        [(20, 10), (1, 1)],
+    ]
+)
+def sizes(request):
+    return request.param
+
+
+@pytest.fixture(params=["h5ad", "zarr"])
+def diskfmt(request):
+    return request.param
+
+
+@pytest.fixture
+def adata(sizes):
+    import dask.array as da
+
+    (M, N), chunks = sizes
+    X = da.random.random((M, N), chunks=chunks)
+    obs = pd.DataFrame(
+        dict(batch=da.array(["a", "b"])[da.random.randint(0, 2, M)]),
+        index=[f"cell{i:03d}" for i in range(M)],
+    )
```

Comment:
```suggestion
    obs = pd.DataFrame(
        {"batch": np.random.choice(["a", "b"], M)},
        index=[f"cell{i:03d}" for i in range(M)],
    )
```

The dask array becomes an in-memory array here